content/ja/handbook/engineering/devops/create/index.md

```
@@ -0,0 +1,326 @@
+---
+
+## こんにちは
+
+[私たちは Create ステージです](/handbook/engineering/devops/create/engineering-managers/)。DevOps 部門内のチームグループです。[GitLab プロダクト内](/handbook/product/categories/#create-stage)の3つの領域で構成されています。
+
+| チーム | エンジニアリングマネージャー |
+| ---- | -------------------- |
+| [Create ステージ](/handbook/engineering/devops/create/) | [André Luís](/handbook/company/team/#andr3) |
+| [Create:Code Review](/handbook/engineering/devops/create/code-review/) | [François Rosé](/handbook/company/team/#francoisrose) |
+| [Create:Source Code](/handbook/engineering/devops/create/source-code/) | [Andre Richards](/handbook/company/team/#andrevr)（バックエンド & フロントエンド） |
+| [Create:Import](/handbook/engineering/devops/create/import/) | [Thiago Figueiró](/handbook/company/team/#thiagocsf)（フルスタック）— 
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)